reports/management/commands/sync_enrolment_reports.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+from reports.services import sync_enrolment_reports
+
+
+class Command(BaseCommand):
+    help = (
+        "Synchronize enrolment reports from enrolment table. "
+        + "Latest sync time will be determined from "
+        + "updated_at -field in a enrolment_reports table. "
+        + "Enrolments after that date will be updated "
+        + "if the enrolment status has changed "
+        + "and created if they are totally missing from the table content."
+    )
+
+    def add_arguments(self, parser):
+        # Named (optional) arguments
+        parser.add_argument(
+            "--sync_from",
+            nargs=1,
+            type=str,
+            help="Determine the date from where to start the sync",
+        )
+        parser.add_argument(
+            "--ignore_linkedevents",
+            action="store_true",
+            help="Determine the date from where to start the sync",
+        )
+
+    def handle(self, *args, **options):
+        sync_from = None
+        if options["sync_from"]:
+            sync_from = datetime.fromisoformat(options["sync_from"][0])
```

Comment:
Django also has `parse_datetime` in `django.utils.dateparse`. Not sure if it has any added benefits thought.

Reply:
I'll try it later. Had to fight a bit to find the solution with datetime.fromisoformat, but now it seems to work ok! 😄